src/render/decorations/IHyprWindowDecoration.cpp

```diff
@@ -26,4 +26,8 @@ eDecorationLayer IHyprWindowDecoration::getDecorationLayer() {
 
 uint64_t IHyprWindowDecoration::getDecorationFlags() {
     return 0;
-}
\ No newline at end of file
+}
+
+std::string IHyprWindowDecoration::getDisplayName() {
+    return std::string("Generic Decoration");
```

Comment:
Unknown would be better. Also no need for `std::string()`

Reply:
done